edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -103,6 +103,8 @@ public static class Constants
 
         public const string EdgeAgentServiceAccountName = "ServiceAccountName";
 
+        public const string EdgeletWorkloadApiVersion = "2018-06-28";
```

Comment:
Since we weren't launching Edge Agent from iotedged in preview, we needed this. However, for release, this should be passed in from iotedged.